src/Slider.js

```diff
@@ -221,68 +298,123 @@ export default class Slider extends PureComponent {
       maximumValue,
       minimumTrackTintColor,
       maximumTrackTintColor,
+      trackColor,
+      trackHighlightColor,
       thumbTintColor,
-      thumbImage,
       styles,
       style,
       trackStyle,
       thumbStyle,
       debugTouchArea,
       ...other
     } = this.props;
+
     var {value, containerSize, trackSize, thumbSize, allMeasured} = this.state;
+
     var mainStyles = styles || defaultStyles;
-    var thumbLeft = value.interpolate({
+
+    let thumbValues = this.state.interpolatedThumbValues = value.map(v => v.interpolate({
       inputRange: [minimumValue, maximumValue],
       outputRange: [0, containerSize.width - thumbSize.width],
       //extrapolate: 'clamp',
-    });
+    }));
+
     var valueVisibleStyle = {};
     if (!allMeasured) {
       valueVisibleStyle.opacity = 0;
     }
 
-    var minimumTrackStyle = {
-      position: 'absolute',
-      width: Animated.add(thumbLeft, thumbSize.width / 2),
-      backgroundColor: minimumTrackTintColor,
-      ...valueVisibleStyle
-    };
-
     var touchOverflowStyle = this._getTouchOverflowStyle();
 
-    return (
-      <View {...other} style={[mainStyles.container, style]} onLayout={this._measureContainer}>
-        <View
-          style={[{backgroundColor: maximumTrackTintColor,}, mainStyles.track, trackStyle]}
-          renderToHardwareTextureAndroid={true}
-          onLayout={this._measureTrack} />
-        <Animated.View
-          renderToHardwareTextureAndroid={true}
-          style={[mainStyles.track, trackStyle, minimumTrackStyle]} />
+    let children = [];
+
+    let trackHighlightStyle;
+
+    if (thumbValues.length > 1) {
+
+      let flatValues = thumbValues.map(x => x.__getValue());
+      this.state.minThumbValue = new Animated.Value(Math.min.apply(Math, flatValues));
+      this.state.maxThumbValue = new Animated.Value(Math.max.apply(Math, flatValues));
+
+      trackHighlightStyle = {
+        position: 'absolute',
+        left: Animated.add(this.state.minThumbValue, thumbSize.width / 2),
+        width: Animated.add(
+          Animated.multiply(this.state.minThumbValue, -1), 
+          this.state.maxThumbValue
+        ),
+        marginTop: -trackSize.height,
+        backgroundColor: minimumTrackTintColor || trackHighlightColor,
+        ...valueVisibleStyle
+      };
+
+    } else {
+      trackHighlightStyle = {
+        position: 'absolute',
+        left: 0,
+        width: Animated.add(thumbLeft, thumbSize.width / 2),
```

Comment:
@danielgindi should the `thumbLeft` be something else given that it no longer exists?

Reply:
Yeah, fixed :-)